src/styles.scss

```diff
@@ -1,5 +1,7 @@
 /* You can add global styles to this file, and also import other style files */
 @import '~@angular/material/prebuilt-themes/deeppurple-amber.css';
+@import '../node_modules/font-awesome/scss/font-awesome.scss';
+$fa-font-path: '../node_modules/font-awesome/fonts';
```

Comment:
why do we need that variable?

Reply:
It was needed due to issues with importing files that have their own relative paths, but it worked without. So i just forgot to delete it. 
Now is removed